nixos/modules/tasks/filesystems/bcachefs.nix

```diff
@@ -0,0 +1,24 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
+
+let
+
+  inInitrd = any (fs: fs == "bcachefs") config.boot.initrd.supportedFilesystems;
+
+in
+
+{
+  config = mkIf (any (fs: fs == "bcachefs") config.boot.supportedFilesystems) {
+
+    system.fsPackages = [ pkgs.bcachefs-tools ];
```

Comment:
that would be nice indeed.

i will look into it!

reference for myself: https://nixos.org/nixos/manual/index.html#sec-building-cd